.github/workflows/security.yml

```diff
@@ -0,0 +1,40 @@
+name: "Security"
+on:
+  workflow_dispatch:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  analyze:
+    name: Security check
+    runs-on: ubuntu-latest
+    env:
+      GO111MODULE: on
+    steps:
+      - name: Checkout Source
+        uses: actions/checkout@v2
+      - name: Run Gosec Security Scanner
+        uses: securego/gosec@v2.9.3
+        with:
+          args: ./...
+  vuln-scan:
+    name: Vulnerability scanner
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+      - name: Run go list
+        run: |
+          if [ -n "$(go mod graph | grep 'github.com/prometheus/common@v0.4.1 github.com/gogo/protobuf@v1.1.1')" ]; then \
```

Comment:
Shouldn't we do the same in `Makefile`? Or this is just a cheating of Nancy?

Reply:
It's just a cheating of Nancy.